composer.json

```diff
@@ -42,7 +45,7 @@
             "Bcgov\\Script\\Standards::phpcbf"
         ],
         "test": [
-            "Bcgov\\Script\\Tests::phpunit"
+            "vendor/bin/phpunit --configuration phpunit.xml.dist"
```

Comment:
i'm assuming we are updating this because how we have change testing, if this is something that is common, maybe we should be bringing back to wordpress-scrips aka wordpress-utils